ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -485,6 +485,32 @@ private static void ApplyHeaders(HttpRequestMessage request, string[][] headers)
                 var key = header[0];
                 switch (key.ToLowerInvariant())
                 {
+                     case "authorization":
+                        //create scheme from user's Authorization first word
+                        string createScheme = header[1].Split(' ')[0];
+
+                        //Remove above added scheme from authorization because it's already added
+                        string completeToken = header[1].Replace(header[1].Split(' ')[0], "");
+
+                        //To create HttpCredentialHedaerValue scheme should not be empty 
+                        if (createScheme.Trim().Length > 0 && completeToken.Contains("\""))
```

Comment:
Is there a better way to detect if the header is in RFC 2617 format than just checking for the string splitting on whitespace? That seems like it may not be the best approach.

Reply:
It has only check for double quotes. And if there is double quotes in the header with equal to operator or :, It removes double quotes.  After the check we are just creating same header so it will not break anything.
In my opinion this is the best check for now. because it's RFC 2617 or generic schema will have application schema key name in start which has separated by space with other token have token keys with  =/: separator.